editor.planx.uk/src/ui/public/InputLabel.tsx

```diff
@@ -13,9 +13,10 @@ export default function InputLabel(props: {
   children: ReactNode;
   hidden?: boolean;
   htmlFor?: string;
+  id?: string;
 }) {
   return (
-    <Root htmlFor={props.htmlFor}>
+    <Root htmlFor={props.htmlFor} id={props.id}>
```

Comment:
This was necessary due to how MUI handles linking a label to the `<Select/>` component.

Helpfully, without this fix, the a11y unit test was failing 🧪 

As this is our first user facing `Select` I think there's likely some work in getting this to work as expected with our existing components and styles perfectly - what I have here is a little rough and ready, but will be revisited once it's wired up to a form.

Docs: https://mui.com/material-ui/react-select/#accessibility